workspaces/client/app/routes/dashboard/settings/internal/components/profile-image-cropper.tsx

```diff
@@ -0,0 +1,222 @@
+		});
+
+	function onImageLoad(e: SyntheticEvent<HTMLImageElement>) {
+		setIsLoaded(true);
+		if (aspect) {
+			const { width, height } = e.currentTarget;
+			setCrop(centerAspectCrop(width, height, aspect));
+		}
+	}
+
+	function onCropComplete(crop: PixelCrop) {
+		if (imgRef.current && crop.width && crop.height) {
+			const croppedImageUrl = getCroppedImg(imgRef.current, crop);
+			setCroppedImageUrl(croppedImageUrl);
+		}
+	}
+
+	function getCroppedImg(image: HTMLImageElement, crop: PixelCrop): string {
+		const canvas = document.createElement("canvas");
+		const scaleX = image.naturalWidth / image.width;
+		const scaleY = image.naturalHeight / image.height;
+
+		canvas.width = crop.width * scaleX;
+		canvas.height = crop.height * scaleY;
+
+		const ctx = canvas.getContext("2d");
+
+		if (ctx) {
+			ctx.imageSmoothingEnabled = false;
+
+			ctx.drawImage(
+				image,
+				crop.x * scaleX,
+				crop.y * scaleY,
+				crop.width * scaleX,
+				crop.height * scaleY,
+				0,
+				0,
+				crop.width * scaleX,
+				crop.height * scaleY,
+			);
+		}
+
+		return canvas.toDataURL("image/png", 1.0);
+	}
+
+	async function onCrop() {
+		try {
+			const blob = await fetch(croppedImageUrl).then((r) => r.blob());
+			const file = new File([blob], `${displayName}.png`, {
+				type: "image/png",
+			});
+			uploadProfileImage({ file });
+		} catch (error) {
+			alert("Something went wrong!");
+		}
+	}
+
+	const handleCancel = useCallback(() => {
+		setDialogOpen(false);
+	}, [setDialogOpen]);
+
+	return (
+		<Dialog
+			title="プロフィール画像の編集"
+			isOpen={dialogOpen}
+			onOpenChange={setDialogOpen}
+		>
+			<div
+				className={css({
+					marginBottom: 4,
+					maxWidth: "500px",
+					width: "100%",
+					aspectRatio: "1 / 1",
+					display: "flex",
+					justifyContent: "center",
+					alignItems: "center",
+				})}
+			>
+				<ReactCrop
+					circularCrop
+					crop={crop}
+					onChange={(_, percentCrop) => setCrop(percentCrop)}
+					onComplete={(c) => onCropComplete(c)}
+					aspect={aspect}
+					className={css({ width: "100%" })}
+				>
+					<img
+						ref={imgRef}
+						className={css({ width: "100%", opacity: isLoaded ? 1 : 0 })}
+						alt="Profile Editor"
+						src={selectedFile?.preview}
+						onLoad={onImageLoad}
+					/>
+					{!isLoaded && (
+						<div
+							className={css({
+								position: "absolute",
+								top: "50%",
+								left: "50%",
+								transform: "translate(-50%, -50%)",
+								color: "gray.500",
+							})}
+						>
+							Loading...
+						</div>
+					)}
+				</ReactCrop>
+			</div>
+			{isUploadError && (
+				<p
+					className={css({
+						color: "red.600",
+						fontSize: "sm",
+						marginBottom: 4,
+					})}
+				>
+					アップロードに失敗しました。切り抜き後の画像が5MiBを超えている可能性があります。
+				</p>
+			)}
+			<div
+				className={css({
+					display: "flex",
+					justifyContent: "flex-end",
+					gap: 4,
+				})}
+			>
+				<button type="button" onClick={handleCancel}>
+					<ButtonLike variant="secondary">Cancel</ButtonLike>
```

Comment:
```suggestion
					<ButtonLike variant="secondary">キャンセル</ButtonLike>
```
「更新」が日本語なので細かいけど気になってしまった